sirplatform.h

```diff
@@ -145,6 +158,16 @@ int pthread_getname_np(pthread_t thread, char* buffer, size_t length);
 #  include <conio.h>
 #  include <shlwapi.h>
 #  include <direct.h>
+#  if defined(__MINGW32__) || defined(__MINGW64__)
+#   define __USE_MINGW_ANSI_STDIO 1
+typedef  /* Workaround a MinGW bug */
+void (__cdecl* _invalid_parameter_handler)(
+ wchar_t const*, wchar_t const*, wchar_t const*,
+ unsigned int, uintptr_t);
+_invalid_parameter_handler
```

Comment:
'sup with dis?

Reply:
MINGW64 provides _invalid_parameter_handler (via UCRT) but does not have a prototype for it (in any header.)